enterprise/app/trends/percentile_chart.tsx

```diff
@@ -0,0 +1,80 @@
+import React from "react";
+
+import * as format from "../../../app/format/format";
+import { ResponsiveContainer, ComposedChart, CartesianGrid, XAxis, YAxis, Line, Legend, Tooltip } from "recharts";
+
+interface Props {
+  title: string;
+  data: any[];
+  extractLabel: (datum: any) => string;
+  formatHoverLabel: (datum: any) => string;
+  extractP50: (datum: any) => number;
+  extractP75: (datum: any) => number;
+  extractP90: (datum: any) => number;
+  extractP95: (datum: any) => number;
+  extractP99: (datum: any) => number;
```

Comment:
can these `any` types be replaced by a stronger type?

Reply:
Done